gridfs.go

```diff
@@ -411,7 +432,7 @@ func (file *GridFile) ContentType() string {
 	return file.doc.ContentType
 }
 
-// ContentType changes the optional file content type.  An empty string may be
+// SetContentType changes the optional file content type.  An empty string may be
```

Comment:
This is a breaking change for users of GridFile - I agree this is what the name should be though.

Opinions @globalsign ?

Reply:
The method name was already `SetContentType`, this is just a documentation change

Reply:
Doesn't look like an issue

Reply:
My bad, thought it was the method that changed